change UI message for sealed to 'OpenBao' (#1652)
openbao/openbao: changelog/1652.txt

```release-note:improvement

> UI shows 'OpenBao is sealed' instead 'Vaul is sealed' for HTTP status 503

Reviewer: changelog entry should have a appropriate category prefix, in this case `ui`
```suggestion
ui: Message changed to 'OpenBao is sealed' from 'Vault is sealed' for HTTP status 503.
```
